odoo/addons/base/tests/test_overrides.py

```diff
@@ -0,0 +1,59 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.exceptions import UserError
+from odoo.tests import TransactionCase, tagged
+
+
+@tagged('-at_install', 'post_install')
+class TestOverrides(TransactionCase):
+
+    # Ensure all main ORM methods behavior works fine even on empty recordset
+    # and that their returned value(s) follow the expected format.
+
+    def test_creates(self):
+        for model in self.env:
+            model_env = self.env[model]
+            if model_env._abstract:
+                continue
+            # with self.assertQueryCount(0):
+            self.assertEqual(
+                model_env.create([]), model_env.browse(),
+                "Invalid create return value for model %s" % model_env._name)
+
+    def test_writes(self):
+        for model in self.env:
+            model_env = self.env[model]
+            if model_env._abstract:
+                continue
+            try:
+                # with self.assertQueryCount(0):
+                self.assertEqual(
+                    model_env.browse().write({}), True,
+                    "Invalid write return value for model %s" % model_env._name)
+            except UserError:
+                # skip models that should never be modified
```

Comment:
best if you could make a list of "to be skipped" modules and to skip them instead of try/except

Reply:
Well, there is a bunch of them, and I don't want to force people to modify this list here and there.
UserError are custom Odoo errors, so I believe discarding those ones for now is acceptable :thinking: